test/core/rules/test_cases/L027.yml

```diff
@@ -26,7 +26,7 @@ test_ignore_value_table_functions:
   pass_str: |
     select
         a.*,
-        _t_start
+        _t_start.*
```

Comment:
This test is a good illustration of the change I made in #722. BigQuery has this odd thing called "value tables", where the alias on line 32 looks like a table alias but can be used as a column. For more details, see the description on #722.